features/taxation/managing_tax_rates/adding_tax_rate.feature

```diff
@@ -9,7 +9,7 @@ Feature: Adding a new tax rate
         And the store has a tax category "Food and Beverage"
         And I am logged in as an administrator
 
-    @ui
+    @api @ui
```

Comment:
```patch
--- @api @ui
+++ @ui @api
```

Discussion took place on slack, that uniformed tags usage should be implemented due spontaneous eye bleeding phenomenon :) 

https://sylius-devs.slack.com/archives/C5Q1H5SNT/p1669745026471029